lib/src/tag/_specs.dart

```diff
@@ -154,36 +154,39 @@ class Platform {
 
   static final android = Platform(
     'Android',
-    Runtime.broadNative,
+    dartRuntime: Runtime.nativeAot,
+    flutterRuntime: Runtime.flutterNative,
     tag: 'platform:android',
   );
   static final ios = Platform(
     'iOS',
-    Runtime.broadNative,
+    dartRuntime: Runtime.nativeAot,
```

Comment:
There is no `dartRuntime` on this platform, so what does this mean?

Reply:
Hm, good point, maybe we make it nullable and leave it out here?